### PR TITLE
Fix #88: Latest download URL

### LINKED ISSUE #88: NGrok Expiration Notice
Got this from NGROK

> We’re dropping you this end-of-year note because we’ve seen you connect to ngrok in the past 90 days with a soon-to-be-deprecated agent. We need to inform you that all ngrok agents version 3.1 and older will stop working on January 15th, 2024 for all free users. 

 

> Your account name XXXX associated to account ID XXXXX last used agent version `2.3.41` on 12/8/2023 14:21:40.

> As per the [ngrok Agent Version Support Policy](https://d2v8tf04.na1.hubspotlinks.com/Ctc/5F+113/d2v8tf04/VW302f3MPK42W83cB5D7lNqDWW6phD5r570F8QN5LPbT23qn9gW7Y8-PT6lZ3q7W8Xnn-n2M3M3vW46sp389gNTjdW578ByQ4w-0S3W5VVCGW2yjFD3W1dd3z050DPNkW4nPrFD5mjzfBW1KDDbN3ywWj0W1d-8rp6lq0LYW3sb6FX6G79xrW19_ys_5-Dcf7W99lYg58bzJ45W4SWYVc72NM-kW4_bs7g8bK188W8LtTjm64dj2-W9dn86n6H84vMW673ct-7PtS-xW3-tYw-7NmRxDN6-55bW8cj9YW7-4KXV3M_rf4W3h0mVf3XpHZqW6VDtn74zMPDRW6GZmfP3D4X5GW6thLXZ1llKF6W7B9nBD6wHCkMW5BFBrG8Mygr4W5B8MXW8wzNHkf4QKlbx04), this serves as a notice that we will be officially ending support for all ngrok agent versions prior to 3.2 starting on January 15th, 2024. After January 15th, 2024, free accounts with agents running 3.1.1 and older will no longer connect to the ngrok service, and any endpoints they were serving will return errors and be unavailable. 

 

> Please follow the instructions on the [ngrok downloads page](https://d2v8tf04.na1.hubspotlinks.com/Ctc/5F+113/d2v8tf04/VW302f3MPK42W83cB5D7lNqDWW6phD5r570F8QN5LPbSM3qn9gW7lCdLW6lZ3l6W5VfRNJ8m8YkfVn2pt84HB3HzN725nLR_y6W_W895zyY83G21RW5r2g3Q2-3bVWW65vVVw6dJDMfW7vcrm86DqNwFW2G8jrB5p0-6QW5sG1HC23k5ZfW319NtL8v5TXVW8sHtqs8drTXhW641M6W7Z9HYVW4kCjlX6cCgT_MV4tY3WbyLMW2yQqdd8-YgQJW8ZRbD54-PDYqW9k7N6m16Qc3rW8WKxVB6wcWG5W1M2_F14MPtGqV_p9Ly7f_2qqVtKkwN4Vj04lW84237T97K2JCW24JkZT3cHpN4W4ylBrF4hNW60dgRLqC04) to upgrade your agent to the latest version before January 15, 2023.

 

ngrok 3.2 will continue to be supported until May 13th, 2024.

### PR DESCRIPTION
**Pull request by melloware**
Fix #88: Latest download URL

### CODE REVIEW

**Comment by melloware** [comment]
@geoand any chance this can be merged and we can do a new release?

**Review by geoand — APPROVED** [review]
Thanks!

**Comment by geoand** [comment]
Would you like to kick off a release?

File: deployment/src/main/java/io/quarkiverse/ngrok/deployment/NgrokDevModeListener.java [hunk 1]
```diff
@@ -26,7 +26,7 @@ public class NgrokDevModeListener implements DevModeListener {
 
     private static final Logger log = Logger.getLogger(NgrokDevModeListener.class);
 
-    private static final String VERSION_STRING = "4VmDzA7iaHb";
+    private static final String VERSION_STRING = "bNyj1mQVY4c";
```
Comment (gsmet) [inline deployment/src/main/java/io/quarkiverse/ngrok/deployment/NgrokDevModeListener.java:29]: Nice version scheme :)

Comment (melloware) [reply deployment/src/main/java/io/quarkiverse/ngrok/deployment/NgrokDevModeListener.java:29]: I know right!

**Pull request merged**